modules/facilities_api/app/services/facilities_api/v2/lighthouse/response.rb

```diff
@@ -40,6 +40,26 @@ def facility
           V2::Lighthouse::Facility.new(data)
         end
 
+        # services is a string here
+        def facility_with_services(services)
```

Comment:
what do you think about this?  I feel that it's a little bit more readable:

```
        def facility_with_services(services)
          facility = V2::Lighthouse::Facility.new(data)
          parsed_services = parse_services(services)
          health_services = extract_health_services(parsed_services)
          facility.access = build_access_data(health_services)
          facility
        end

        private

        def parse_services(services)
          JSON.parse(services).fetch('data', [])
        end

        def extract_health_services(services_data)
          health_services = services_data.map do |service|
            V2::Lighthouse::Service.new(service)
          end

          health_services.select { |service| service['new'] || service['established'] }
        end

        def build_access_data(health_services)
          if health_services.empty?
            { 'health' => [], 'effectiveDate' => '' }
          else
            { 'health' => health_services, 'effectiveDate' => health_services.first['effectiveDate'] }
          end
        end

```

Reply:
Definitely more readable.

Reply:
Accepted changes. Tests pass.